android/src/main/java/com/it_nomads/fluttersecurestorage/ciphers/RSACipher18Implementation.java

```diff
@@ -110,8 +110,16 @@ private void createRSAKeysIfNeeded(Context context) throws Exception {
         KeyStore ks = KeyStore.getInstance(KEYSTORE_PROVIDER_ANDROID);
         ks.load(null);
 
-        Key privateKey = ks.getKey(KEY_ALIAS, null);
-        if (privateKey == null) {
+        // see:
+        // https://stackoverflow.com/questions/36488219/android-security-keystoreexception-invalid-key-blob
+        KeyStore.Entry entry = null;
+        try {
+            entry = ks.getEntry(KEY_ALIAS, null);
```

Comment:
Hi @prprhyt,
thanks for MR.

I have 2 questions:
* according to the SO link, why don't you add retry?
* if we delete KEY_ALIAS, it means we are not able to decode existing data from prefs. correct?